cpu/native/include/async_read.h

```diff
@@ -58,6 +58,15 @@ void native_async_read_cleanup(void);
 void native_async_read_continue(int fd);
 
 /**
+ * @brief   helper for read continue with select
+ *
+ * Call this function after reading file descriptors.
+ *
+ * @param[in] fd  The file descriptor to monitor
+ */
```

Comment:
I guess the documentation could be improved a bit (grammar in `@brief`, purpose, side-effect, "monitor" might be a bit confusing).